test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,33 @@
+require "test_helper"
+
+class UsersControllerTest < ActionController::TestCase
+  setup do
+    login_user(users(:john))
+  end
+
+  test "GET index should be successfull" do
+    get :index
+    assert_response :success
+  end
+
+  test "POST create should create a new user" do
+    assert_difference "User.count" do
+      xhr :post, :create, user: { email: "example@mail.org" }
```

Comment:
Maybe you also want to check if the email has been sent here. (Or at least the UserMailer class got called with the right params)